lib/msf/ui/console/command_dispatcher/db.rb

```diff
@@ -312,27 +312,18 @@ def add_host_tag(rws, tag_name)
 
     rws.each do |rw|
       rw.each do |ip|
-        opts[:ip] = ip
+        opts[:address] = ip
         framework.db.add_host_tag(opts)
       end
     end
   end
 
-  def find_hosts_with_tag(workspace_id, host_address, tag_name)
```

Comment:
why was this piece of functionality removed?

Reply:
It's not shown in these changes, but existing code in the [hosts](https://github.com/rapid7/metasploit-framework/blob/fc164e0c098456bcc65d3f2605a0580ff0e2965e/lib/msf/core/db_manager/host.rb#L144) method can already search for hosts with any passed tags.